src/inputs/checkbox.jsx

```diff
@@ -1,13 +1,22 @@
-import React, { Component } from 'react';
-import _ from 'lodash';
+import {
+    isFunction,
+} from 'lodash';
+import React from 'react';
 import ClassNames from 'classnames';
 import PropTypes from 'prop-types';
 import Icon from '../dataDisplay/icon';
 
 const propTypes = {
-    align: PropTypes.oneOf([ 'left', 'right' ]),
-    checked: PropTypes.bool,
+    align: PropTypes.oneOf(['left', 'right']),
+    checked: false,
```

Comment:
We did we change from `PropTypes.bool` to `false` here?

Reply:
Thank you for catching this.

Reply:
Fixing.

Reply:
Commit 7c8b691 fixes this.

Reply:
Yep, looks good.  Resolving.